python/lsst/afw/table/base/baseContinued.py

```diff
@@ -21,15 +21,15 @@
 #
 from __future__ import absolute_import, division, print_function
 
-__all__ = ["Catalog"]
-
 from future.utils import with_metaclass
 
 import numpy as np
 
 from lsst.utils import continueClass, TemplateMeta
 from .base import BaseRecord, BaseCatalog
 
+__all__ = ["Catalog"]
+
```

Comment:
Is this something that's been announced to the LSST community? The last I'd heard about this issue I'd gotten the impression that it was the guidelines that were not PEP8-compliant.